src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -74,8 +76,8 @@ public Serializable getDefaultValue( final IdentifyingAttributes comp, final Str
 
 	public boolean isDefault( final String tag, final String attribute, final String attributeValue ) {
 		final String defaultValue = getDefaultValue( tag, attribute );
-		if ( defaultValue != null ) {
-			return defaultValue.equalsIgnoreCase( attributeValue );
+		if ( StringUtils.equalsIgnoreCase( defaultValue, attributeValue ) ) {
```

Comment:
How about moving it behind the `null` check for `attributeValue` and then to:

```java
if ( attributeValue.equalsIgnoreCase( defaultValue ) ) { ... }
```

Reply:
do you mean something like that: 


`if ( attributeValue == null || attributeValue.trim().isEmpty() ) {`
    `return true;`
`}`

`if ( attributeValue.equalsIgnoreCase( defaultValue ) {`
    ` return true;`
`}`